src/BSD-3-Clause.xml

```diff
@@ -28,21 +29,21 @@
             </item>
             <item>
                 <bullet>3.</bullet>
-                <alt match="(The name of.+may not)|(Neither the name of.+nor the names of its contributors may)|( Neither the name of.+nor the names of its authors and contributors may)|(Neither the name of.+nor the names of contributors may)|(Neither the name.+nor the names of contributors may)|(The names of its contributors may not)|(The names of any contributors may not)|(None of the names of.+and any contributors may)" name="organizationClause3">
+                <alt match="(The name of.+may not)|(Neither the name of.+nor the names of its contributors may)|( Neither the name of.+nor the names of its authors and contributors may)|(Neither the name of.+nor the names of contributors may)|(Neither the name.+nor the names of contributors may)|(The names of its contributors may not)|(The names of any contributors may not)|((The names of the contributors may not)|None of the names of.+and any contributors may)" name="organizationClause3">
                  Neither the name of the copyright holder nor the names of its contributors may</alt>
                  be used to endorse or promote products derived from this
                <alt match="software|work" name="software">software</alt> without specific prior written permission.
             </item>
         </list>
-        <p>THIS SOFTWARE IS PROVIDED <alt match=".*" name="copyrightHolderAsIs">BY THE COPYRIGHT HOLDERS AND
+        <p>THIS <alt match="(SOFTWARE)|(THEME)" name="software">SOFTWARE</alt> IS PROVIDED <alt match=".*" name="copyrightHolderAsIs">BY THE COPYRIGHT HOLDERS AND
```

Comment:
`software` is already used as the name for the preceding alt tag, so this one should be `software2`. I believe these are expected to be different for each alt match (probably something we need to fix in the CI checks if it isn't being picked up).